<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hexxagon</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>982</width>
        <height>640</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGra">
    <property name="title">
     <string>Gra</string>
    </property>
    <addaction name="actionNowa_Gra"/>
    <addaction name="actionZako_cz"/>
   </widget>
   <addaction name="menuGra"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNowa_Gra">
   <property name="text">
    <string>Nowa Gra</string>
   </property>
  </action>
  <action name="actionZako_cz">
   <property name="text">
    <string>Zako≈Ñcz</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
